Add "Return YouTube Dislike" support and add video information to TV UI

Outlet7493 opened the pull request:
* Add integration for https://returnyoutubedislike.com/ to all clients - this is off by default and can be enabled in settings - addresses https://github.com/lamarios/clipious/issues/17 if desired
* Add video info (views, likes, and publish text) to TV UI
- [x] I have updated the version and build number accordingly

lamarios commented:
Thanks ! I appreciate it. Since it is my first feature PR from someone I was wondering, is the code easy to understand and update ?

lamarios reviewed (changes requested):
Tested it and it works great so far, only request is that we move the new setting in the browsing category instead of video player.

Outlet7493 commented:
Apologies, moved it to the browsing category now.

I added this feature with relative ease and it was pretty simple to find anything I did need! :smile:

lamarios commented:
Thanks a lot !
